src/renderer/components/select/select.tsx

```diff
@@ -65,8 +66,10 @@ export class Select extends React.Component<SelectProps> {
     makeObservable(this);
   }
 
-  @computed get theme() {
-    return this.props.themeName || ThemeStore.getInstance().activeTheme.type;
+  @computed get themeClass() {
```

Comment:
Why did we change `Select`? Out of scope for this PR, right?

Reply:
There is one more bug with `<Select>` still (which I found while writing this PR). That when used within `<Wizard>` the background colour for "Selected and hover" is still the global theme.